{{cookiecutter.project_slug}}/README.md

```diff
@@ -51,9 +53,18 @@ mbed toolchain {{cookiecutter.toolchain}}
 
 {% else -%}
 Enable the custom target:
-```shell
-cp {{ cookiecutter.__custom_target_repo }}/custom_targets.json .
-```
+
+* Linux:
```

Comment:
Using `*` is discouraged and does not respect the other documents we produce in md format. Unordered lists should use `-`.

This kind of specificities between Linux and Windows platforms are why I think we should remove this section.